### PR TITLE
Minor README.md updates

### PR DESCRIPTION
**Pull request by sferrini**
Just some minor README updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +16,7 @@ If you are passionate about iOS and kernel exploitation and want to help us make
 
 ---
 To start the process we first need to prepare a kernel image, a device tree, a static trust cache, the main and the secondary disk images.
-To get the all that we first need to get the update file from Apple: [iOS 12.1 update file].
+To get the all that, we first need to get the update file from Apple: [iOS 12.1 update file].
```
Comment (aronsky) [inline README.md]: Thanks for noticing. Let's also remove the `the`, which shouldn't be here in the first place (To get ~~all~~ that)

File: README.md [hunk 2]
```diff
@@ -49,23 +47,27 @@ $ python3 xnu-qemu-arm64-tools/bootstrap_scripts/asn1dtredecode.py Firmware/all_
 
 ## Create the Disk Devices for iOS system
 
-Some tweaks should be done to use the all currently implemented capabilities; bash, many familiar binary tools, all iOS's launchd services, r/w secondary disk device and SSH. 
+Some tweaks should be done to use the all currently implemented capabilities; bash, many familiar binary tools, all iOS's launchd services, r/w secondary disk device and SSH.
```
Comment (aronsky) [inline README.md]: * [...] to use ~~the~~ all currently [...]
* let's change ';' to ':'

**Comment by sferrini** [comment]
@aronsky oh I missed those :) just pushed the fix.

**Comment by aronsky** [comment]
Thanks!

**Pull request merged**